app/views/admin/schedules/show.html.haml

```diff
@@ -1,48 +1,36 @@
-.schedule-content
-  %h2
-    = "Schedule for #{@conference.title}"
-  #unscheduled.unscheduled
-    .unscheduled-header
-      Unscheduled Events
-  #schedule.schedule
-    .schedule-dates-header
-      %ul#date-tabs
-        - @dates.each do |date|
-          %li.date-selector
-            = link_to "#{date}", "#", id: "#{date}-selector", onclick: "Schedule.changeDay('#{date}')"
-    .schedule-rooms-container
-      = render 'day_tab'
+.row
+  .col-md-12
+    .page-header
+      %h1 Schedule
+      %p.text-muted
+        Create the schedules for the conference
+
+.row
+  .col-md-2
+    Selected schedule
+    = check_box_tag @conference.short_title, @schedule.id, (@schedule.id == @selected_schedule.try(:id)),
+      method: :patch, url: (admin_conference_program_path(@conference.short_title) + '?[program][selected_schedule_id]='),
```

Comment:
Yes, and it doesn't run the migration twice ... 
We would need to restart the heroku server or you run it locally!